src/components/faqTemplates/ExpandableWithCategories.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { SectionContainer } from '@lib/Containers'
+import { Spacers } from '@lib/Helpers'
+
+const FaqAnswerContainer = styled.div`
+  display: ${props => (props.shouldDisplay ? 'block' : 'none')};
```

Comment:
shall we use `p`?
```suggestion
  display: ${p=> (p.shouldDisplay ? 'block' : 'none')};
```

Reply:
> How are we differentiating between `.js` and `.jsx`?

jsx files will indicate React components and js will indicate files without React components.

Reply:
that's a good idea. `props -> p`

Reply:
Are you implying only the styled-components should change `props -> p` or both react components and styled-components should change `props ->`

Reply:
mainly styled components I think, react components up to you but I would spread the props if there aren't that many props